openedx/core/lib/tests/test_time_zone_utils.py

```diff
@@ -0,0 +1,95 @@
+"""Tests covering time zone utilities."""
+from django.test import TestCase
```

Comment:
Yes, it should.

If this code needs django, for any reason, it should move to `openedx/core/djangolib` instead.